packages/workflow/src/interfaces.ts

```diff
@@ -66,10 +66,33 @@ export interface WorkflowInfo {
    *
    * This value changes during the lifetime of an Execution.
    *
+   * When this value is zero the history length is unknown.
```

Comment:
Scratch that. It is not possible for `historyLength` to ever be unknown/zero in WorkflowInfo (that is only possible on client side, specifically in response to `list()`).

Reply:
Done